include/db/generic.inc.php

```diff
@@ -250,7 +250,7 @@ function serendipity_db_schema_import($query) {
             static $search  = array('{AUTOINCREMENT}', '{PRIMARY}',
                 '{UNSIGNED}', '{FULLTEXT}', '{FULLTEXT_MYSQL}', '{BOOLEAN}', '{TEXT}');
             static $replace = array('int(11) not null auto_increment', 'primary key',
-                'unsigned'  , 'FULLTEXT', 'FULLTEXT', 'enum (\'true\', \'false\') NOT NULL default \'true\'', 'LONGTEXT');
+                'unsigned'  , 'FULLTEXT', 'FULLTEXT', 'enum (\'true\', \'false\')', 'LONGTEXT');
```

Comment:
I don't think we can do this. Plugins and parts of the core this commit missed might rely both on the NOT NULL and on the default value.

I assume you made that change because the `subscribe` table should have default false? In that case I propose to add a `{BOOLEAN_FALSE}` to the schema instead of changing what's there.

Reply:
We can do that safely. Your objection is valid, I didn't think of the additional plugins, where it is used in event_photoblog. I changed it there accordingly to `{BOOLEAN} NOT NULL DEFAULT 'false' ` if serendipity core < 2.4. No other occurrence of this element in the complete code basis. For the comments table, I did that already in db.sql with this commit.

Reply:
When core > 2.4, right?

The thing is: We don't know all plugins. While yes, probably everything is on spartacus, but there are some custom installations and plugins out there. Changing the core like that means we need to announce it, give time, bump the serendipity version to 3.x with the next release. That's okay if it has a big positive impact, but here I don't see the need.

If we just introduce {BOOLEAN_FALSE} we can have the subscription feature without all of this.